content/guides/getstarted/hello-world-in-your-browser.md

```diff
@@ -273,48 +255,73 @@ console.log("Transaction sent with hash:", txHash);
 
 ### Run the application
 
-With the client application written, you can run the code via the same `run`
-command.
+Now that you know how the client works, you can run the code via the `run`
+command. Just type it in the playground terminal and press enter.
+
+```shell
+run
+```
 
 Once your application completes, you will see output similar to this:
 
 ```shell
+$ run
 Running client...
   client.ts:
-    My address: GkxZRRNPfaUfL9XdYVfKF3rWjMcj5md6b6mpRoWpURwP
-    My balance: 5.7254472 SOL
-    Sending transaction...
-    Transaction sent with hash: 2Ra7D9JoqeNsax9HmNq6MB4qWtKPGcLwoqQ27mPYsPFh3h8wignvKB2mWZVvdzCyTnp7CEZhfg2cEpbavib9mCcq
+    ProgramID:  C7JcX81YDaDJ9Bf8ebifLgBSqfKJxETek6qyTuPGJE1f
+    Transaction sent with hash: m7f7dszzdNshMZo5u2gRKjbyURk1tQHj7Hmeh3AbH7wUdnmDXmCJhA8cXJjYwVN7foJaLQiPYhEFTn6F5mWSeCb
+    Congratulations! Look at your transaction in the Solana Explorer:
+  https://explorer.solana.com/tx/m7f7dszzdNshMZo5u2gRKjbyURk1tQHj7Hmeh3AbH7wUdnmDXmCJhA8cXJjYwVN7foJaLQiPYhEFTn6F5mWSeCb?cluster=devnet
 ```
 
-### Get transaction logs
+You can now click on the link provided to view your transaction in the Solana
+Explorer. If you scroll to the bottom you should see `Hello, world!` in the
+**Log Messages** section of the program output. 🎉
 
-We will be using `solana-cli` directly in playground to get the information
-about any transaction:
+#### Congratulations!!!
 
-```shell
-solana confirm -v <TRANSACTION_HASH>
-```
+You have now written a Solana program, deployed it to devnet and called the
+program. You are now a Solana developer!
 
-Change `<TRANSACTION_HASH>` with the hash you received from calling
-`hello world` program.
+PS: Try to update your program's message then re-build, re-deploy, and
+re-execute your program.
 
-You should see `Hello, world!` in the **Log Messages** section of the output. 🎉
+### Look at the deployed program
 
-#### Congratulations!!!
+Now that we called our program let's have a look at the account where the
+program is actually saved.
 
-You have now written a client application for your onchain program. You are now
-a Solana developer!
+When executing a program using [web3.js](/docs/clients/javascript-reference.md)
+or from [another Solana program](/docs/core/cpi.md), you will need to provide
```

Comment:
```suggestion
When executing a program using [web3.js](/docs/clients/javascript-reference)
or from [another Solana program](/docs/core/cpi), you will need to provide
```

These work even if you add `.md`, so maybe you want to keep them, but the actual URLs don't have the `.md`

Reply:
Is there any difference between the link with .md or with?

Reply:
having the .md extension in the file makes most code editors use auto complete. And will allow people to click on them when directly browing the markdown files on github.

I strongly recommend using them

Reply:
Okey, ill leave it as is then

Reply:
I also just updated the contributing guide to state this too. I thought I had put it in there before, but I guess not